packages/message-parser/src/utils.ts

```diff
@@ -85,14 +85,17 @@ export const link = (src: string, label?: Markup[]): Link => ({
   value: { src: plain(src), label: label ?? [plain(src)] },
 });
 
+export const isValidCustomDomain=(domain:string,customDomains?: string[])=>{
+  return customDomains && customDomains.includes(domain);
+}
+
```

Comment:
Hey rique223! Thanks for your review, this does make sense to avoid unnecessary linting and syntax errors. It was my first time raising a PR here, I will read the links provided by you to get a better understanding of the code structure followed here. For now I am committing your suggestion.

I was also trying to generate a changeset. This will be a PATCH I assume, right? Not a MAJOR or MINOR bump?

Reply:
> This will be a PATCH I assume, right? Not a MAJOR or MINOR bump?

No, this will be a MINOR as you've added new functionality to the system.

Reply:
Thanks rique223! I have added the changeset. Do let me know if anything else is required.